wbstatus.py

```diff
@@ -106,18 +107,20 @@ def get_user(self, phid):
         return retval
 
 
-# the TaskStore is a wrapper around the Phabricator manifest.query
-# API call, so this object indexes the result by task number.  This
-# is necessary because the Phabricator API inexplicably doesn't
-# return the result in such a way that the tasks can be easily
-# looked up by task number.
 class TaskStore(object):
+    """The TaskStore is a wrapper around the Phabricator manifest.query API
+    call, so this object indexes the result by task number.  This is necessary
+    because the Phabricator API inexplicably doesn't return the result in such
+    a way that the tasks can be easily looked up by task number.
+    """
 
     def __init__(self, tasknums=set()):
         self.tasknums = tasknums
 
     def load_from_phabricator(self, phab, cachedir):
-        taskquerycall = lambda: phab.maniphest.query(ids=self.tasknums)
```

Comment:
Changed back to lambdas in the next patch. I haven't actually run this successfully locally do to a lack of workboard html scrapes.